cmd/serf/command/agent/command.go

```diff
@@ -734,7 +752,10 @@ Options:
                            -bind if the interface is known but not the address.
                            If both are provided, then Serf verifies that the
                            interface has the bind address that is provided. This
-                           flag also sets the multicast device used for -discover.
+                           flag also sets the multicast device used for -discover,
+                           if mdns-iface is not specified.
+  -mdns-iface              Network interface to use for mDNS. If not provided, the
+                           -iface value is used.
```

Comment:
Need doc for the other boolean flags too.

Reply:
Added